star_tides/requirements.txt

```diff
@@ -0,0 +1,11 @@
+flask
+psycopg2-binary
+sqlalchemy
+alembic
```

Comment:
Wasn't this a Postgres thing? Also, is there any way to tell Pip to use certain revisions/SHAs for these deps?

Reply:
That is a postgres thing. Removing now.
```
SomePackage            # latest version
SomePackage==1.0.4     # specific version
SomePackage>=1.0.4'     # minimum version
```